source/auth/auth.rst

```diff
@@ -1159,6 +1175,264 @@ If AWS authentication fails for any reason, the cache MUST be cleared.
 .. note::
     Five minutes was chosen based on the AWS documentation for `IAM roles for EC2 <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html>`_ : "We make new credentials available at least five minutes before the expiration of the old credentials". The intent is to have some buffer between when the driver fetches the credentials and when the server verifies them.
 
+MONGODB-OIDC
+~~~~~~~~~~~~
+
+:since: 7.0 Enterprise
+
+MONGODB-OIDC authenticates using an `OIDC <https://openid.net/specs/openid-connect-core-1_0.html>`_ access tokens.  Drivers MUST support
+both the Authentication Code Flow and Device Workflow for AWS, Azure,
```

Comment:
This sounds a bit like drivers would need to implement Authentication Code Flow out of the box, when in reality that’s left to user-provided callbacks (which may be populated with example code for ACF) – maybe this could be reworded to clarify this?

Reply:
Good catch, updated